app/models/atmosphere/action.rb

```diff
@@ -0,0 +1,13 @@
+module Atmosphere
+  class Action < ActiveRecord::Base
```

Comment:
You mean "future" i guess?... A relation to Appliance Type definitely can be implemented i n the future just as a nullable reference to any other entity. I don't understand what you mean by PR